RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# Release To The Google Play Store
+
+## Setup
+
+Run the setup script:
+
+```
+> bin/setup
```

Comment:
Drop the `> `? Maybe make it `./bin/setup`? http://stackoverflow.com/questions/6331075/why-do-you-need-dot-slash-before-script-name-to-run-it-in-bash